code/game/mecha/equipment/tools/other_tools.dm

```diff
@@ -367,8 +367,7 @@
 	return ..()
 
 /obj/item/mecha_parts/mecha_equipment/generator/proc/generator_init()
-	fuel = new /obj/item/stack/sheet/mineral/plasma(src)
-	fuel.amount = 0
+	fuel = new /obj/item/stack/sheet/mineral/plasma(src, 0)
```

Comment:
I think this and the other instances where `0` is passed will not pass the `if (new_amount)` check and the stack will effectively start with one sheet instead. Easiest fix would be to change that check to `if(new_amount != null)`.

Reply:
Good catch, there appears to be a problem with action buttons tooltips on the latest byond beta so I couldn't test those.